src/Model/Result/IResult.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Model.Result;
+
+public interface IResult
+{
+    ResultType ResultType {get;}
+    string QuestionResult {get; set;}
+    int userId {get;}
+    int questionId {get;}
+    int surveyId {get;}
```

Comment:
You use camel case for these variables in the interface but pascal case in the implementation
```suggestion
    int UserId {get;}
    int QuestionId {get;}
    int SurveyId {get;}
```